app/javascript/components/ExpressDonation/ExpressDonation.js

```diff
@@ -279,7 +245,7 @@ export class ExpressDonation extends Component {
             />
           </div>
 
-          {/* {!this.state.recurringDonar && (
+          {/* {!this.state.recurringDonor && (
```

Comment:
Could you remove this left over commented block? Also if there's other stuff in the file that's just commented out, if you could remove it that'd be great.

Reply:
Done.